lib/Search/ElasticSearch/ElasticSearchIndexer.php

```diff
@@ -454,28 +464,11 @@ private function indexBatch($module, array $beans)
      *
      * @return bool
      */
-    private function differentialIndexing()
+    private function differentialIndexing(): bool
     {
         return $this->differentialIndexing && $this->lastRunTimestamp !== false;
     }
 
-    /**
-     * Retrieves the default params to set up an optimised default index for Elasticsearch.
-     *
-     * @return array
-     */
-    private function getDefaultMapParams()
```

Comment:
Hi @Dillon-Brown, the reason for no longer needing the map params is related with elastic search 7?

Reply:
Yup exactly, ElasticSearch will now reject anything that comes with mappings like this.